client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx

```diff
@@ -248,12 +250,13 @@ const ClinicalDetails: React.FC = (): JSX.Element => {
                             <DatePick
                                 test-id='symptomsStartDate'
                                 type='date'
-                                value={(!isUnkonwnDateChecked && context.clinicalDetailsData.symptomsStartDate !== null) ? format(context.clinicalDetailsData.symptomsStartDate as Date, dateFormatForDatePicker) : dateFormatForDatePicker}
+                                defaultValue={(!isUnkonwnDateChecked && context.clinicalDetailsData.symptomsStartDate !== null) ? format(context.clinicalDetailsData.symptomsStartDate as Date, dateFormatForDatePicker) : dateFormatForDatePicker}
                                 lableText='תאריך התחלת סימפטומים'
                                 disabled={isUnkonwnDateChecked}
-                                onChange={(event: React.ChangeEvent<HTMLInputElement>) => (
-                                    updateClinicalDetails(ClinicalDetailsFields.SYMPTOMS_START_DATE, new Date(event.target.value))
-                                )}
+                                onBlur={(event: React.FocusEvent<HTMLInputElement>) => {
+                                    const newDate = event.target.value ? new Date(event.target.value) : null
+                                    updateClinicalDetails(ClinicalDetailsFields.SYMPTOMS_START_DATE, newDate)
+                                }}
```

Comment:
maybe pass to func, or better add newDate handling inside the Datepicker component itself so it wouldnt have to be added on every datepicker use?

Reply:
newDate only related when Blur effect is triggered so it cannot be implemented as a prop in date pick,
I don't see the benefit of passing 2 lines to a function when only one of these lines is repetitive.
Not saying no, only asking for a better argument for doing such a minor change

Reply:
date pick could have an internal onBlur func that has 
` const newDate = event.target.value ? new Date(event.target.value) : null` 
inside and then calls external onblur with newDate

Reply:
that would prevent datepicker users from having to know they need to add this line to prevent bugs

Reply:
if using onChange with DatePick is dangerous and might cause this bug to happen then it can be removed from the DatePick props so people wont be able to use it

Reply:
i agree that the move to func suggestion wasnt beneficial 🤷‍♀️